test/webdriverio/test/test_setup.ts

```diff
@@ -558,6 +558,7 @@ export async function sendKeyAndWait(
   keys: string | string[],
   times = 1,
 ) {
+  // @ts-ignore: Unintentional comparison error
```

Comment:
Instead of this I think you can just declare PAUSE_TIME as the number type:
export const PAUSE_TIME: number = 0;

That should prevent its type from getting set to the literal value and avoid the unintentional comparison error when it's not 0.

Reply:
Alas no: if I do that then eslint complains:
```
  46:14  error  Type number trivially inferred from a number literal, remove type annotation  @typescript-eslint/no-inferrable-types
```
I could disable eslint to allow the declaration, but it seems more straightforward to disable it at the point the ts/lint errors actually occur:
```JS
  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
  // @ts-ignore: Unintentional comparison error
  if (PAUSE_TIME === 0) {
    // …
```

So many well-intentioned rules making it so difficult to just change the value of a numeric constant!